internal/locate/region_request.go

```diff
@@ -1767,10 +1812,16 @@ func regionErrorToLabel(e *errorpb.Error) string {
 		return "flashback_not_prepared"
 	} else if e.GetIsWitness() != nil {
 		return "peer_is_witness"
+	} else if isDeadlineExceeded(e) {
```

Comment:
Maybe we need to distinguish the default `KvTimeout` and `ConfigurableKvTimeout` errors.
@you06  What do you think?